src/tactic/lift.lean

```diff
@@ -108,7 +108,7 @@ do
     fail "lift tactic failed. Tactic is only applicable when the target is a proposition.",
   e ← i_to_expr p,
   old_tp ← infer_type e,
-  new_tp ← i_to_expr t,
+  new_tp ← i_to_expr ``(%%t : Type*),
```

Comment:
```suggestion
  new_tp ← i_to_expr ``(%%t : Sort*),
```
Unless you deliberately avoided this

Reply:
`can_lift` is currently only defined for `Type*`. It could potentially be useful on `Sort*`. Do you think I should generalize it?

Reply:
My thought was mainly that this would make result in a less weird error if someone does end up passing a Sort rather than a Type; the coercion would still be inserted, and then the error would come later. But I'm only guessing, and this is hardly a big deal.

I don't know enough to know if the generalization of `can_lift` is interesting.